### ISSUE TITLE
Stubgen produces <TypeList ...> instead of [...]

### ISSUE THREAD

**Issue by evhub** [python/mypy#4640]
Under `mypy 0.560`, running `stubgen` on
```python
import typing

f: typing.Callable[[int, int], int]
```
produces
```python
# Stubs for prelude (Python 3.6)
#
# NOTE: This dynamically typed stub was automatically generated by stubgen.

import typing

f: typing.Callable[<TypeList int, int>, int]
```
which is invalid Python syntax. Instead, `<TypeList int, int>` should be `[int, int]`.

**Comment by evhub** [comment]
Whoops, closing as a duplicate of #4638.

**Comment by evhub** [comment]
Reopening since #4638 was closed as a duplicate. This is fixed by PR #4650.
